README.md

```diff
@@ -4,9 +4,13 @@
 
 [![Build Status](https://travis-ci.org/ubclaunchpad/minutes.svg?branch=master)](https://travis-ci.org/ubclaunchpad/minutes)
 
-Audio speaker diarization and transcription API.
+[![Coverage Status](https://coveralls.io/repos/github/ubclaunchpad/minutes/badge.svg)](https://coveralls.io/github/ubclaunchpad/minutes)
 
-## :running: Getting Started
+Audio speaker diarization and transcription library.
```

Comment:
Are we going to ship transcription with Minutes as well? It'll complicate the library a bit, since we'd probably want to call out to Google Cloud Speech-to-Text (unless there's a nice offline library that does it for us).

Reply:
Good call, I'll remove this.